pkg/alertmanager/api_grafana.go

```diff
@@ -290,6 +291,8 @@ func (am *MultitenantAlertmanager) GetUserGrafanaConfig(w http.ResponseWriter, r
 			Hash:                      cfg.Hash,
 			CreatedAt:                 cfg.CreatedAtTimestamp,
 			Default:                   cfg.Default,
+			Promoted:                  cfg.Promoted,
```

Comment:
Quick fix, we forgot to add this field in the response to `GET api/v1/grafana/config`